### PR TITLE
Curve chart and rules block

### PR DESCRIPTION
**Pull request by MostafaBalata**
screen shot 2017-10-26 at 12 26 11 pm

### CODE REVIEW

File: app/components/CurveChart.tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+const getCumulativeNEU = (
+  currencyRate: number,
+  initialReward: number,
+  capNEU: number,
+  atETH: number
+) => {
+  const atEUR = atETH / currencyRate;
+  return (1 - Math.exp(-1 * initialReward * atEUR / capNEU)) * capNEU;
+};
+*/
```
Comment (MostafaBalata) [inline app/components/CurveChart.tsx]: Marcin said comment this function, don't remove it.

File: app/components/PriceCalculator.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        </li>
+        <li>
+          Finish: <b>4.25</b> NEU / 1 EUR
+        </li>
+        <br />
+        <li>Post-ICO phase:</li>
+        <li>
+          Start price: <b>4.25</b> NEU / 1 EUR
+        </li>
+        <li>Finish: ...</li>
```
Comment (MostafaBalata) [inline app/components/PriceCalculator.tsx]: I'm not sure yet what is this data represent

File: webpack.config.js [hunk 3]
```diff
@@ -59,8 +59,8 @@ module.exports = {
     }
   },
   entry: {
-    main: [...devEntryPoints, "./app/index.tsx"],
-    commit: [...devEntryPoints, "./app/commit.tsx"],
+    main: [...devEntryPoints, "./app/index.tsx", "./app/curve.tsx", ],
```
Comment (krzkaczor) [inline webpack.config.js]: 👍

File: app/components/CurveChart.tsx [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+  currentRasiedEther?: number;
+}
+
+export default (props: ICurveChart) => {
+  const { min, max, dotsNumber, capNEU, initialReward, currencyRate, currentRasiedEther } = props;
+  neuMarkInfoCurveChartPlugin();
+
+  const etherDatasetList = getEtherDataset(min, max, dotsNumber);
+
+  let activePointIndex = 0;
```
Comment (krzkaczor) [inline app/components/CurveChart.tsx]: use lodash `findIndex` https://lodash.com/docs/4.17.4#findIndex

Comment (krzkaczor) [inline app/components/CurveChart.tsx]: what will happeen if corrent raised ether is somewhere between datapoints?

**Pull request merged**